src/components/textbox/textbox-autocomplete/textbox-autocomplete.tsx

```diff
@@ -1,64 +1,64 @@
 /** @jsx h */
```

Comment:
I only changed one line in file but the file was not prettierfied before so it now looks like a big change.